homework/App.js

```diff
@@ -11,14 +11,89 @@ class App {
    * Initialization
    * @param {string} url The GitHub URL for obtaining the organization's repositories.
    */
-  async initialize(url) {
-    // Add code here to initialize your app
-    // 1. Create the fixed HTML elements of your page
-    // 2. Make an initial XMLHttpRequest using Util.fetchJSON() to populate your <select> element
 
+  async initialize(url) {
     const root = document.getElementById('root');
+    Util.createAndAppend('h1', root, { text: 'REPOSITORES' });
```

Comment:
I can't pass you on this part. You need to use the classes `Contributor` and `Repository` with the appropriate render methods, this is important to understand before the react module. Let me know if you need any assistance with this!

Reply:
Alright :) I have watched a bunch of tutorials on this and hope to finally to be able to solve it especially since we got even more additional guidance in the class. It takes a while to find out how these all .js files are connected, but once I understand it, I hope to be able to use it over and over again in the future projects. My main issue is being able to put apart and use small parts of the huge functions without damaging the functionality.